servers/zms/src/main/java/com/yahoo/athenz/zms/ZMSImpl.java

```diff
@@ -5099,17 +5099,24 @@ List<Policy> setupPolicyList(AthenzDomain domain, Boolean assertions, Boolean ve
                             .setName(policy.getName())
                             .setModified(policy.getModified())
                             .setVersion(policy.getVersion())
-                            .setActive(policy.getActive());
+                            .setActive(policy.getActive())
+                            .setTags(policy.getTags());
                     policies.add(newPolicy);
                 }
             }
         }
 
+        if (tagKey != null) {
+            policies = policies.stream()
+                    .filter(policy -> (filterByTag(tagKey, tagValue, policy, Policy::getTags) && ((versions == Boolean.TRUE) || policy.getActive() == Boolean.TRUE)))
```

Comment:
why are we including 

((versions == Boolean.TRUE) || policy.getActive() == Boolean.TRUE)

in the filter. Shouldn't we be only looking at the tag key?

Reply:
The previous condition was:
```
if (versions != Boolean.TRUE && policy.getActive() == Boolean.FALSE) {
                    continue;
}
```
That means we would like to collect only active policies or if the request wants to get all policies.
If tag key/value exists, we need to add on top of that (only if the previous condition accepted) tag key-value filtering.

Reply:
sorry you are right. I reviewed it locally and missed that..

> in the filter. Shouldn't we be only looking at the tag key?
We probably want to filter by tag value also .. correct?